__tests__/homePage.test.jsx

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react'
+import Home from '@/app/page'
+
+describe('Page', () => {
+	test('Home page renders an h1', () => {
```

Comment:
todo: In React component tests, we often focus on the final DOM. So in this case, it could be a single tests with 3 assertions:

```js
describe('Homepage', () => {
	it('renders correctly', () => {
		render(<Home />)
		const title = screen.getByRole('heading', { level: 1, name: 'Scrum Diving' })
		expect(title).toBeInTheDocument()

		const subheader = screen.getByRole('heading', {
			level: 2,
			name: "Because it's not really poker",
		})
		expect(subheader).toBeInTheDocument()

		const link = screen.getByRole('link', { name: 'Host a Room' })
		expect(link).toBeInTheDocument()
	})
})
```

Reply:
Okay, I'll do that!!